core/app/models/spree/option_type.rb

```diff
@@ -21,10 +21,11 @@ class OptionType < Spree::Base
 
     accepts_nested_attributes_for :option_values, reject_if: lambda { |ov| ov[:name].blank? || ov[:presentation].blank? }, allow_destroy: true
 
+    after_save :touch
```

Comment:
This seems weird. Doesn't that just trigger another, unnecessary update.

Can we either:
a) remove it?
b) if that doesn't work just do `after_save touch_all_products`?